FIRBoundaries.dat

```diff
@@ -3080,60 +3080,59 @@ EFIN|0|0|80|59.000000|19.086667|70.116667|31.583333|63.733333|27.000000
 68.379275|22.827508
 68.4712|22.134828
 68.765556|21.379722
-68.765556|21.379722
 69.022228|22.094528
-68.783333|22.5
-68.7|23.283333
-68.766667|23.633333
-68.866667|23.733333
-68.816667|24.216667
-68.766667|24.3
-68.65|24.916667
-68.833333|25.133333
-69.016667|25.566667
+68.783372|22.498583
+68.700042|23.281961
+68.766714|23.631975
+68.866719|23.731975
+68.816719|24.215336
+68.766719|24.298678
+68.650053|24.915381
+68.833394|25.13205
+69.016739|25.565394
 69.693889|25.952778
 69.695556|29.133333
-69.216667|28.983333
+69.216772|28.982233
 69.051944|28.928611
-68.916667|28.45
+68.916667|28.450000
 68.883333|28.533333
-68.866667|28.8
-68.55|28.45
-68.2|28.666667
-68.066667|29.35
-67.8|29.683333
+68.866667|28.800000
+68.550000|28.450000
+68.200000|28.666667
+68.066667|29.350000
+67.800000|29.683333
 67.666667|30.016667
-67.516667|29.95
+67.516667|29.950000

```

Comment:
Could you check these changes? in a few cases you're just adding zero's while we just added a PR to round them all correctly

Reply:
Fixed in c1764d049cf9be535b1aa8ea006c3ca360d65d91. Removed other commit